README.md

```diff
@@ -1870,6 +1871,22 @@ Enable STARTSSL. Defaults to `false`.
 
 The name of the mailbox where incoming mail will end up. Defaults to `inbox`.
 
+##### `IMAP_METHOD`
+
+Method blank or microsoft_graph
+
+##### `IMAP_TENANT_ID`
+
+Azure AD Tenant ID
+
+##### `IMAP_CLIENT_ID`
+
+Azure AD App Client ID
+
+##### `IMAP_CLIENT_SECRET`
+
+Azure AD App Client Secret
+
```

Comment:
As with the other parameters, it may be useful to explain the default values ​​here (All the values added this time are "No default," though)

```suggestion
##### `IMAP_METHOD`

If you want to use Microsoft Graph API, set to `microsoft_graph`. No default

##### `IMAP_TENANT_ID`

Azure AD Tenant ID. No default

##### `IMAP_CLIENT_ID`

Azure AD App Client ID. No default

##### `IMAP_CLIENT_SECRET`

Azure AD App Client Secret. No default

```